color_r_python_css.Rmd

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
I'm not sure if this is ok, but, to be safe, please remove this empty line.

![1668376550278](https://user-images.githubusercontent.com/114460912/201546478-269908e0-94e9-4733-be16-3a06b11f1a48.png)